locations/categories.py

```diff
@@ -368,13 +377,13 @@ def apply_category(category, item: Feature):
     "leisure",
     "man_made",
     "office",
+    "power",
     "public_transport",
     "shop",
+    "telecom",
     "tourism",
-    "aeroway",
```

Comment:
why have you removed this?

Reply:
Just reordered alphabetically :) It's still in the list of top level tags.